### ISSUE TITLE
[Feature Request] Chips: Change mouse icon to pointer when @click() is present

### ISSUE THREAD

**Issue by danielschmitz** [vuetifyjs/vuetify#4687]
When:

```
<v-chip @click="onChipClick()">........
```

The mouse pointer should be

![image](https://user-images.githubusercontent.com/1509692/43047899-2523709c-8db5-11e8-9df7-1cc1d8498b19.png)

Thanks! Awesome framework!!

**Comment by jsmedmar** [comment]
I agree, some how using style doesn't work:

```css
style="cursor: pointer"
```

**Comment by marcelogft** [comment]
This is working for me (using 1.3.11):
 
```  <v-icon large color="blue darken-2" style="cursor: pointer;">fab fa-facebook-square</v-icon> ```

**Comment by blalan05** [comment]
This is available in 2.0 alpha.

**Comment by chovy** [comment]
doesn't work. using 2.2.20
